### PR TITLE
LIP0045: fix minor issues

### PR DESCRIPTION
**Pull request by ricott1**
Address issues in https://github.com/LiskHQ/lips/issues/216 and internal review.

### CODE REVIEW

**Review by AndreasKendziorra — CHANGES_REQUESTED** [review]

File: proposals/lip-0045.md [hunk 1]
```diff
@@ -970,7 +966,7 @@ def bounce(ccm: CCM, newCCMStatus: uint32, ccmProcessedEventCode: uint32) -> Non
 The `createTerminatedStateAccount` function creates an entry in the terminated state substore.
 
 ```python
-def createTerminatedStateAccount(chainID: ChainID, stateRoot: bytes = EMPTY_BYTES) -> None:    
+def createTerminatedStateAccount(chainID: ChainID, stateRoot: bytes = EMPTY_HASH) -> None:
```
[the remark this answers is not in the document]
Comment (ricott1) [reply proposals/lip-0045.md:969]: fixed

File: proposals/lip-0045.md [hunk 2]
```diff
@@ -558,22 +556,22 @@ ownChainAccountSchema = {
...
 On the manchain, the own chain account is present by default, set to an object with properties:
 
-* `name=CHAIN_NAME_MAINCHAIN`,
-* `ID=getMainchainID()`,
-* `nonce=0`,
+* `name = CHAIN_NAME_MAINCHAIN`,
+* `ID = getMainchainID()`,
+* `nonce = 0`,
 
 serialized with the JSON schema `ownChainAccountSchema`.
```
[the remark this answers is not in the document]
Comment (ricott1) [reply proposals/lip-0045.md:573]: true, but maybe it's fine to keep it for extra clarity, just writing the full properties

[the remark this answers is not in the document]
Comment (AndreasKendziorra) [reply proposals/lip-0045.md:573]: ok

**Review by AndreasKendziorra — APPROVED** [review]

File: proposals/lip-0045.md [hunk 3]
```diff
@@ -1390,7 +1380,11 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
   * The `bftWeight` property of each element must be positive integer.
   * Let `totalWeight` be the sum of the `bftWeight` property of every element in `activeValidators`. Then `totalWeight` has to be less than or equal to `MAX_UINT64`.
   * Check that `totalWeight//3 + 1 <= certificateThreshold <= totalWeight`, where `//` indicates integer division.
-* If a chain account for another sidechain is present, than a chain account for the mainchain (`chainID == getMainchainID()`) must be present, as well as the own chain account.
+  * Check that the corresponding `validatorsHash` stored in `chainAccount.lastCertificate.validatorsHash` matches with the value computed from `activeValidators` and `certificateThreshold`.
+* If a chain account for another sidechain is present (`storeKey != getMainchainID()`), then a chain account for the mainchain (`storeKey == getMainchainID()`) must be present, as well as the own chain account.
```
Comment (gkoumout) [inline proposals/lip-0045.md:1384]: This makes sense only for sidechains. State this explicitly.

Also I guess for sidechains the `registeredNamesSubstore` property should be empty or even do not exist in the `genesisInteroperabilityStoreSchema`.

**Review by gkoumout — APPROVED** [review]
Left a small comment, but can be approved from my side.

File: proposals/lip-0045.md (hunk 3 above)
Comment (ricott1) [reply proposals/lip-0045.md]: added

**Review by gkoumout — APPROVED** [review]

**Review by janhack — APPROVED** [review]

**Pull request merged**